api/agent_test.go

```diff
@@ -1070,15 +1070,16 @@ func TestAPI_AgentForceLeave(t *testing.T) {
 	}
 }
 
-func TestAPI_AgentForceLeavePrune(t *testing.T) {
+func TestAPI_AgentForceLeaveWithOpts(t *testing.T) {
 	t.Parallel()
 	c, s := makeClient(t)
 	defer s.Stop()
 
 	agent := c.Agent()
 
 	// Eject somebody
-	err := agent.ForceLeavePrune("foo")
+	q := &QueryOptions{Datacenter: "", Prune: true}
```

Comment:
Also maybe it's worth to add test for `forceLeaveWithOpts` with datacenter specified

Reply:
Yep, that would be useful.